### PR TITLE
use interfaceOrientation for ios 13+

### PR DESCRIPTION
**Pull request by parveshneedhoo**

### CODE REVIEW

File: src/ios/CameraSessionManager.m [hunk 1]
```diff
@@ -15,7 +15,14 @@ - (CameraSessionManager *)init {
 }
 
 - (AVCaptureVideoOrientation) getCurrentOrientation {
-    return [self getCurrentOrientation: [[UIApplication sharedApplication] statusBarOrientation]];
+    UIInterfaceOrientation orientation;
+    if (@available(iOS 13.0, *)) {
+        UIWindowScene *activeWindow = (UIWindowScene *)[[[UIApplication sharedApplication] windows] firstObject];
+        orientation = [activeWindow interfaceOrientation] ?: UIInterfaceOrientationPortrait;
+    } else {
+        orientation= [UIApplication sharedApplication].statusBarOrientation;
```
Comment (zfir) [inline src/ios/CameraSessionManager.m:23]:
```suggestion
        orientation = [UIApplication sharedApplication].statusBarOrientation;
```

**Review by zfir — APPROVED** [review]

File: src/ios/CameraRenderController.m [hunk 2]
```diff
@@ -46,7 +46,13 @@ - (void) viewWillAppear:(BOOL)animated {
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(appplicationIsActive:) name:UIApplicationDidBecomeActiveNotification object:nil];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(applicationEnteredForeground:) name:UIApplicationWillEnterForegroundNotification object:nil];
     
-    UIInterfaceOrientation orientation= [UIApplication sharedApplication].statusBarOrientation;
+    UIInterfaceOrientation orientation;
```
[the remark this answers is not in the document]
Comment (dinitri) [reply src/ios/CameraRenderController.m:49]: @parveshneedhoo like @ombr said we should avoid duplication for this logic.
@ombr we first refactor, and then decide which version of iOS to drop or not (investigate which versions are no more used)

**Review by ombr — APPROVED** [review]
Code looks good, I did not test

**Review by dinitri — APPROVED** [review]
tested, looks good, you can make an npm versoin

**Pull request merged**